tests/original_prm/original.prm

```diff
@@ -1,10 +1,93 @@
 # test that output/original.prm is written correctly
 
 # based on no_flow.prm:
-include $ASPECT_SOURCE_DIR/tests/no_flow.prm
```

Comment:
Just an idea for the future: we could automatically add a comment in the original.prm that shows the include statement (or something like "# including content of <xyz>:")